libs/template/src/lib/let/let.directive.ts

```diff
@@ -77,7 +66,8 @@ export interface LetViewContext<T> extends RxViewContext<T>  {
  * - triggers change-detection differently if ViewEngine or Ivy is present (`ChangeDetectorRef.detectChanges` or
  *   `ɵdetectChanges`)
  * - distinct same values in a row (`distinctUntilChanged` operator),
- * - display custom templates for different observable notifications (suspense, next, error, complete)
+ * - display custom templates for different observable notifications (rxSuspense, next, error, complete)
```

Comment:
```suggestion
 * - display custom templates for different observable notifications (rxSuspense, rxNext, rxError, rxComplete)
```
Wouldn't this be more consistent of a description?